### PR TITLE
Age and Sex Prediction #86

### PR DESCRIPTION
**Pull request by ASHISHKUMAR2411**
# Pull Request for DL-Simplified 💡

## Issue Title : Age and Sex Prediction #86

- **Info about the related issue (Aim of the project)**: The objective of the project is to Create a DL-based model which can detect gender and age using facial images. Convolutional Neural Network is used to classify the images. There are 2 output types namely, gender(M or F) and age.
- **Name:** Ashish Kumar
- **GitHub ID:** https://github.com/ASHISHKUMAR2411
- **Email ID:** ashishkumar24112001@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)** SSOC - 2022

Closes: #86

### Describe the add-ons or changes you've made 📃

In the notebook, I trained a pre-trained CNN model using the above dataset and adjusted the algorithm. It also uses Convolutional Layers from Convolutional Neural Networks and compares the accuracy for age and gender and it also figures out the loss during model training. We have trained the model with a large variety of images that covers pose, facial expression, illumination,  occlusion, resolution, etc so that we can enhance the accuracy of the model.

## Type of change ☑️

What sort of change have you made:

- [x] My Code adds a new feature through which we can detect the age and sex of the person's through its image. 

## How Has This Been Tested? ⚙️

Describe how it has been tested
I have tested my code with many test images, and also include those tests in the code, which give a perfect accuracy of more than 95%

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
I have gone through the whole project, it's really a well presented one. But you need to implement more models (total count should be atleast 3, to get a proper comparison), to find out the best fitted one. Using a single model we can't figure out the best fitted model for this project as there is no place of comparison. Hope you are getting my concern.

@ASHISHKUMAR2411

**Comment by ASHISHKUMAR2411** [comment]
Okay, I will try to implement the other two as well, thank you.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Put your new `.ipynb` file inside the project folder, do not put that outside the project folder. You have created a sub folder of Models, put the models/ipynb files inside that.

Update the README.md file, and state what are the models you have used and also provide a comparison of them using the accuracy scores.
@ASHISHKUMAR2411

**Review by abhisheks008 — APPROVED** [review]
Looks good to me. Congratulations on your first contribution in Deep Learning Simplified project repo.

Your PR is approved and ready to be merged 💯 
@ASHISHKUMAR2411
